tests/unit/test_utils.py

```
@@ -58,10 +58,10 @@ def test_causes():
         try:
             # e3 will be e2's __cause__
             raise e2 from e3
-        except Exception:
+        except Exception:  # noqa: BLE001
             # e2 will be e1's __context__
             raise e1
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)